azure-pipelines.yml

```diff
@@ -0,0 +1,60 @@
+name: das-forecasting-jobs
+trigger:
+  batch: true
+  branches:
+    include:
+      - "*"
+
+pr: none
+
+workspace:
+    clean: all
+
+variables:
+  BuildConfiguration: 'Release'
+  Build.MajorNumber: '1'
+  Build.MinorNumber: '0'
```

Comment:
As above, use the Gitversion task for build number versioning and remove these variables